_stbt/types.py

```diff
@@ -13,6 +14,19 @@ class Size(namedtuple('Size', 'width height')):
     """Size of a rectangle with ``width`` and ``height``."""
 
 
+class Direction(Enum):
+
+    #: Process the image from left to right
+    HORIZONTAL = "horizontal"
```

Comment:
Do these need values?

Reply:
How would you define an enum without them?